packages/arco-lib/src/components/Table.tsx

```diff
@@ -21,6 +27,7 @@ import {
 import { TableInstance } from '@arco-design/web-react/es/Table/table';
 
 const TableStateSchema = Type.Object({
+  clickedRow:Type.Optional(Type.Any()),
```

Comment:
I think we don't have to distinguish `clickedRow` and `selectedRow`. We just need to select the row after clicking it. Does arco support manually select a row?